python/Produce_Config_File.py

```diff
@@ -86,24 +86,23 @@ def CyclicColor(DrawArg):
 	text_file.write("\t\t\tPlot_Name = '%s';\n"%(str(legEntry)))
 	text_file.write("\t\t\t[BEGIN_DATA]\n" )
 	text_file.write("\t\t\tVAR_INDEP,VAR_DEP,VAR_INDEP_ERR,VAR_DEP_ERR\n" )
-	DrawArg=DrawArg+1
 
 	for row in range(args.SelectRowStart,args.SelectRowEnd):
-		if bool(args.YaxisScale):
+		if args.YaxisScale=='true':
```

Comment:
Yes you are right.. the  'true'  is being interpreted as a string 
But as I describe in changelog  I changed all the bool statements to strings in options.py 
So the statement is working fine